scripts/mkcloud

```diff
@@ -774,9 +776,59 @@ function testsetup()
     onadmin testsetup
     local ret=$?
     $scp root@$adminip:tempest.log "$artifacts_dir/tempest.log"
+    $scp root@$adminip:tempest.subunit.log "$artifacts_dir/tempest.subunit.log"
```

Comment:
I don't like this change. I am hiding the real name of both logfiles.  I found this more explicit.